### PR TITLE
Add Chinese(zh-TW) translation of README

### PR DESCRIPTION
**Pull request by NightFeather**

### CODE REVIEW

File: README.zh-TW.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+所有發行的版本都會以 [B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A) 簽署
+並發佈於 [GitHub](https://github.com/swaywm/sway/releases)
+
+## 安裝
+
+### 從套件安裝
+
+Sway 在許多發行版都有提供。請自己嘗試於你的發行版安裝 「sway」這個套件。
+如果無法取得，請查看 [這個 wiki 頁面](https://github.com/swaywm/sway/wiki/Unsupported-packages)
+以取得更多關於如何於你使用的發行板上安裝的資訊。
```
Comment (xdavidwu) [inline README.zh-TW.md]: typo, 板->版

File: README.zh-TW.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+如果你想要為你使用的發行板包裝 sway，請到 IRC 頻道或是直接寄封信到 sir@cmpwn.com 來取得一些建議。
```
Comment (xdavidwu) [inline README.zh-TW.md]: same here, typo, 板->版

File: README.zh-TW.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## 設定檔
+
+如果你已經在使用 i3，你可以直接將你的 i3 設定檔複製到 `~/.config/sway/config` 然後就能直接使用。
+或者你也可以把範例設定檔複製到 `~/.config/sway/config`。 它通常會在 `/etc/sway/config`。
+執行 `man 5 sway` 來取得更多關於設定檔的資訊。
+
+## 執行
+
+在 TTY 執行 `sway`。有些 sway 不支援的 display manager 可能可以運作 (已知 gdm 運作的很好)
```
Comment (xdavidwu) [inline README.zh-TW.md]:
```suggestion
在 TTY 執行 `sway`。有些 display manager 可能可以運作但 sway 不提供支援 (已知 gdm 運作的很好)
```
This would be a better translation.

**Comment by emersion** [comment]
@xdavidwu, can you review again this PR? Thanks

**Comment by emersion** [comment]
Thanks!

@NightFeather Can you rebase?

**Comment by NightFeather** [comment]
Rebased

**Comment by emersion** [comment]
Thanks!

**Pull request merged**